decs.d.ts

```diff
@@ -1,7 +1,7 @@
 declare module '@gridsuite/commons-ui' {
     import { FunctionComponent, ReactElement } from 'react';
     import { AutocompleteProps } from '@mui/material/Autocomplete/Autocomplete';
-    import { RadioGroupProps, SxProps, TextFieldProps } from "@mui/material";
+    import { RadioGroupProps, SxProps, TextFieldProps } from '@mui/material';
```

Comment:
CI crashes if you don't do this changes ?
it would be better to do it in another PR

Reply:
My IDE did it automatically, i removed the unnecessary changes